<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtColorPaletteClass</class>
 <widget class="QMainWindow" name="QtColorPaletteClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtColorPalette</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="RedLabel">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="GreenLabel">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="BlueLabel">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="RedLayout">
          <item>
           <widget class="QSlider" name="RedSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="RedBox">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="GreenLayout">
          <item>
           <widget class="QSlider" name="GreenSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="GreenBox">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="BlueLayout">
          <item>
           <widget class="QSlider" name="BlueSlider">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="BlueBox">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtColorPalette.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BlueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlueBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GreenBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GreenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RedBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
